### PR TITLE
Update contributors.yml

### PR DESCRIPTION
**Pull request by antonina-kyoseva**
Can you add me to the contributors? :)

### CODE REVIEW

**Comment by beyhan** [comment]
Hi @antonina-kyoseva,
could you please provide a small description why?

**Review by boyan-velinov — APPROVED** [review]

**Review by ikasarov — APPROVED** [review]

**Comment by boyan-velinov** [comment]
@emalm  We added 2 approvals (me and Ivan) but still the PR shows **Review required**
What is the next step

**Comment by stephanme** [comment]
Process description: https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md#promotion-to-contributor
Technically, the PR needs a +1 by an approver of the community project (= TOC members and WG leads) before it can be merged.

Next step: @gerg merges this PR or it gets merged in the next TOC meeting.

**Review by AP-Hunt — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Review by beyhan — APPROVED** [review]

**Pull request merged**
